examples/get_event_extra_fields.json

```diff
@@ -1,86 +0,0 @@
-{
-  "products": {
-    "identification": {
-      "data": {
-        "visitorId": "Ibk1527CUFmcnjLwIs4A9",
-        "requestId": "0KSh65EnVoB85JBmloQK",
-        "incognito": true,
-        "linkedId": "somelinkedId",
-        "time": "2019-05-21T16:40:13Z",
-        "timestamp": 1582299576512,
-        "url": "https://www.example.com/login",
-        "ip": "61.127.217.15",
-        "ipLocation": {
-          "accuracyRadius": 10,
-          "latitude": 49.982,
-          "longitude": 36.2566,
-          "postalCode": "61202",
-          "timezone": "Europe/Dusseldorf",
-          "city": {
-            "name": "Dusseldorf"
-          },
-          "continent": {
-            "code": "EU",
-            "name": "Europe"
-          },
-          "country": {
-            "code": "DE",
-            "name": "Germany"
-          },
-          "subdivisions": [
-            {
-              "isoCode": "63",
-              "name": "North Rhine-Westphalia"
-            }
-          ]
-        },
-        "browserDetails": {
-          "browserName": "Chrome",
-          "browserMajorVersion": "74",
-          "browserFullVersion": "74.0.3729",
-          "os": "Windows",
-          "osVersion": "7",
-          "device": "Other",
-          "userAgent": "Mozilla/5.0 (Windows NT 6.1; Win64; x64) ...."
-        },
-        "confidence": {
-          "score": 0.97
-        },
-        "visitorFound": true,
-        "firstSeenAt": {
-          "global": "2022-03-16T11:26:45.362Z",
-          "subscription": "2022-03-16T11:31:01.101Z"
-        },
-        "lastSeenAt": {
-          "global": "2022-03-16T11:28:34.023Z",
-          "subscription": null
-        }
-      }
-    },
-    "botd": {
-      "data": {
-        "bot": {
-          "result": "notDetected"
-        },
-        "url": "https://www.example.com/login",
-        "ip": "61.127.217.15",
-        "time": "2019-05-21T16:40:13Z"
-      }
-    },
-    "product3": {
```

Comment:
We need this example to test our SDKs against new undescribed fields

Reply:
Ah, so it's not used anywhere in this repository but copied over to SDK repositories and used there?

Reply:
That also means I can't change any example file names without also updating every single Server SDK. Should I just revert the name changes?

Reply:
@JuroUhlar could you rename this file too to have a consistent naming?

Reply:
Done, thanks!